### ISSUE TITLE
User manual

### ISSUE THREAD

**Issue by DanieleDeSensi** [PIFO-TM/ns3-bmv2#3]
Hi, is there any user manual on how to use this simulator?

Thanks

**Comment by radhakrishnaKamath** [comment]
I don't think this is anymore maintained. It is not clear how to run this in ns3
